be2-scala/build.sbt

```
@@ -32,10 +36,10 @@ copyProtocolTask := {
     }else{
         val source = new File("../protocol")
         val dest   = new File("./src/main/resources/protocol")
-        Try(IO.copyDirectory(source,dest, true)) match {
+        Try(IO.copyDirectory(source,dest, overwrite = true)) match {
```

```suggestion
        Try(IO.copyDirectory(source, dest, overwrite = true)) match {
```
shouldn't the spacing be something like this?

---

Strange that it was not modified by the automatic linter... 🤔. I couldn't find any other examples of such commas. There might be some, but probably insanely scattered/rare

---

EDIT: turns out IntelliJ didnt change anything because build.sbt is not a scala file